cocos/game/game.ts

```diff
@@ -408,6 +408,19 @@ export class Game extends EventTarget {
         if (this._pacer) this._pacer.targetFrameRate = this._frameRate;
     }
 
+    /**
+     * @en Make engine update in faked fixed interval regardless of the actual running interval.
+     * @zh 让引擎以假的时间间隔更新，而不考虑实际的运行间隔。
+     * @param deltaTime
+     * @en The faked fixed interval, the unit is second. Will not use faked interval if the value is less or equal to 0.
+     * @zh 假的时间间隔，单位是秒。如果传入的值小于或等于0，那么不使用假的时间间隔。
+     * @engineInternal
+     */
+    public setFixedDeltaTime (deltaTime: number) {
+        this._deltaTime = deltaTime;
```

Comment:
I think `useFixedDeltaTime` and `setFixedDeltaTime` should be used as two interfaces. Otherwise, turning off useFixedDeltaTime requires game.setFixedDeltaTime = 0, which is weird!

Reply:
And `_fixedDeltaTime` should be new property? no needs to write `_deltaTime`. 

```
get fixedDeltaTime () {
     return this._fixedDeltaTime;
}
set fixedDeltaTime (val) {
     this._fixedDeltaTime = val;
}

get useFixedDeltaTime () {
     return this._useFixedDeltaTime;
}
set useFixedDeltaTime (val) {
     this._useFixedDeltaTime = val;
}

get deltaTime () {
      return this._useFixedDeltaTime ? this._fixedDeltaTime : this._deltaTime;
}

step () {
       const deltaTime = this._useFixedDeltaTime ? this._fixedDeltaTime : this._deltaTime;
       director.tick(deltaTime);
}
```

Reply:
It looks good. Then should `_useFixedDeltaTime ` be a property too?

Reply:
As there is already `frameTime`, and it seems to be used as fixed delta time, so just reuse it. The the codes don't have to invoke `set useFixedDeltaTime` in order to enable/disable using fixed delta time. If engine is driven by `step()`, then treat it as using fixed delta time.